File: extra3/app/src/main/java/com/example/extra3/db/TaskEntity.java
package com.example.extra3.db;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "tasks")
public class TaskEntity {
    @PrimaryKey(autoGenerate = true)
    public int id;
    public String title;
    public long createdAt;

    public TaskEntity(String title, long createdAt) {
        this.title = title;
        this.createdAt = createdAt;
    }
}

File: extra3/app/src/main/java/com/example/extra3/db/TaskEntity.kt
package com.example.extra3.db

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "tasks")
data class TaskEntity(
    @PrimaryKey(autoGenerate = true) val id: Int = 0,
    val title: String,
    val createdAt: Long = System.currentTimeMillis()
)
